### DISCUSSION TITLE
typo in comment

### DISCUSSION THREAD

**Discussion by wzhcz8902** [NVIDIA/cutlass#1504] [Q&A] [answered]
https://github.com/NVIDIA/cutlass/blob/5c447dd84f8ae0e1d48ff9a2eae26ce8c4958101/include/cutlass/gemm/warp/mma_tensor_op.h#L140-L168

As a newbie to cutlass, I think this struct is targeted for tensor cores, not cuda cores as the comment claims.

**Comment by hwu36** [comment, marked as answer]
yes, you are correct.  copy and paste error.  thanks for reporting it.
